cmd/tink-worker/cmd/root.go

```diff
@@ -59,9 +59,7 @@ func NewRootCommand(version string, logger log.Logger) *cobra.Command {
 
 			ctx := context.Background()
 			if timeOut > 0 {
-				var cancel context.CancelFunc
-				ctx, cancel = context.WithTimeout(ctx, timeOut)
-				defer cancel()
+				logger.Info("timeOut is deprecated")
```

Comment:
lets just get rid of the code, we have enough workarounds/deprecated things and it just turns into baggage. We're still at v <1.0 and also have sandbox to provide a cushion for breakages.

Reply:
sure thing